src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTryFinally.cs

```diff
@@ -61,6 +59,109 @@ .maxstack  0
 }");
         }
 
+        [Fact]
```

Comment:
:bulb: If you use a `[Theory]` here, you could actually write the test so the contents of each block is parameterized as a clean way to validate the code generated for `""` and `";"` is the same.

```csharp
[Theory]
[InlineData("")]
[InlineData(";")]
public void NopInTryCatchFinally(string nopText)
```

Reply:
Hmmm, that's a great idea.